Userland/Applications/PixelPaint/Tools/GradientTool.h

```diff
@@ -32,19 +36,37 @@ class GradientTool : public Tool {
     virtual StringView tool_name() const override { return "Gradient Tool"sv; }
 
 private:
+    enum class GradientMode {
+        Linear,
+        Radial,
+    };
+
+    enum class IconStyle {
+        None,
+        ChangeWidthAndAngle,
+        RadialWith,
+    };
+
     RefPtr<GUI::Widget> m_properties_widget;
     Optional<Gfx::IntPoint> m_gradient_start;
     Optional<Gfx::IntPoint> m_gradient_center;
     Optional<Gfx::IntPoint> m_gradient_end;
+    Optional<Gfx::IntPoint> m_gradient_left;
+    Optional<Gfx::IntPoint> m_gradient_right;
     Gfx::IntPoint m_perpendicular_point;
 
+    GradientMode m_mode = GradientMode::Linear;
+    Vector<DeprecatedString> m_mode_options;
```

Comment:
These options don't need to be a vector of `DeprecatedString`, they're know at compile-time so can be an `Array` of `StringView`, which also does not need to be a member variable (see the `mode_combo` of `PolygonalSelectTool`).